panos_upgrade_assurance/utils.py

```diff
@@ -248,6 +265,18 @@ def _extract_element_name(config: Union[str, dict]) -> str:
         else:
             raise exceptions.WrongDataTypeException("Config definition is neither string or dict")
 
+    def _remove_invalid_requested_config(self, invalid_keys: set) -> None:
```

Comment:
I'm just wondering if it make sense to build a method for 3 lines of code that won't be used anywhere else besides the constructor